ktor-utils/jvm/src/io/ktor/util/reflect/ServiceLoader.kt

```diff
@@ -13,6 +13,9 @@ import java.util.*
 
 /**
  * Loads all implementations of the service [T] using [ServiceLoader.load].
+ *
+ * [Report a problem](https://ktor.io/feedback?fqname=io.ktor.util.reflect.loadServices)
+ *
  * @see loadServiceOrNull
  */
 @InternalAPI
```

Comment:
Should we skip `@InternalApi` declarations from the generator?

Reply:
I think it's ok if they have KDoc written. At the moment, I'm skipping only public without KDocs